backend/user_service/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "demo-service",
+  "version": "1.0.0",
+  "description": "This Service allows users to access jokes",
```

Comment:
Now that i think of it we should probably update this for all our package.json AHAHAHAHA

Reply:
but dont need do in current pr we can create another separate branch to address this